### ISSUE TITLE
How to get position and size of window

### ISSUE THREAD

**Issue by tinymouse** [asticode/go-astilectron#127]
I can set position (X, Y) and size(Height, Width) when I make window using astilectron.WindowOptions.
I want to get position and size when window is closed. How can I get them?

**Comment by asticode** [comment]
@tinymouse right now you can't, but there is a pending PR that would add this feature: https://github.com/asticode/go-astilectron/pull/115

**Comment by tinymouse** [comment]
I see. I feel bad to hear that I cannot. 
I tried to use PR #155, but I found that
w.o.X and w.o.Y do not change when I move the window without using method w.Move(), and 
w.o.Height and w.o.Width do not change when I resize the window without using method w.Resize().

**Comment by asticode** [comment]
yes you're right, some changes are needed in this PR and I'm waiting for them

**Comment by tinymouse** [comment]
Thank you for your comment. I hope that window object has a new method.
